BuildInformation.md

```diff
@@ -0,0 +1,7 @@
+# Quality Tools for BlueJ (QT4BJ) - Build Information
+
+## Required Java version(s)
+Your JDK version should match the requirements of your version of BlueJ. See the [main BlueJ website](https://www.bluej.org/) or [BlueJ installations](https://www.bluej.org/versions.html) to verify that your JDK version is correct
+
+## Gradle wrapper usage
+A Gradle wrapper class is supplied with this project. Simply replace all `gradle` commands with the `gradlew` command to use the wrapper class and ensure the correct build settings are used.
```

Comment:
What does "all `gradle` commands" refer to?

Isn't `gradlew` _"the recommend way to execute any Gradle build"?_ See [Gradle User Guide - Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html)

Reply:
I have re-worded my Gradle statements in my most recent commit